core/main/src/broker/endpoint_broker.rs

```diff
@@ -420,36 +420,44 @@ impl BrokerOutputForwarder {
                         {
                             let request_id = rpc_request.ctx.call_id;
                             v.data.id = Some(request_id);
-                            if let Some(result) = v.data.result.clone() {
-                                if is_event {
-                                    if let Some(filter) = broker_request
-                                        .rule
-                                        .transform
-                                        .get_filter(super::rules_engine::RuleTransformType::Event)
-                                    {
-                                        if let Ok(r) = jq_compile(
-                                            result,
-                                            &filter,
-                                            format!("{}_event", rpc_request.ctx.method),
-                                        ) {
-                                            v.data.result = Some(r);
-                                        }
-                                    }
-                                } else if let Some(filter) = broker_request
+
+                            let jv: Value = if let Some(result) = v.data.clone().result {
```

Comment:
Thunder will return the event as Params
{
method: "id.event"
params: { `payload` }
}

This will not work for all brokers